pom.xml

```diff
@@ -43,6 +43,14 @@
                     <includeTestSourceDirectory>true</includeTestSourceDirectory>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>16</source>
+                    <target>16</target>
+                </configuration>
+            </plugin>
```

Comment:
Why do you need this?

Reply:
IDEA requested me to change java version when i wanted to use toList() instead of Collectors.toList() in ProcessDataImpl. I just click on the recommendation of IDEA😂. And also Bohdan Chupika said that we can change the java version in our project.